compiler/optimizer/LoopVersioner.cpp

```diff
@@ -192,7 +192,7 @@ int32_t TR_LoopVersioner::performWithDominators()
       {
          printf("WARNING: method may have infinite loops\n");
       }
-
+   _postDominators = NULL;
```

Comment:
This isn't a global pointer. It is a field of the `TR_LoopVersioner` class. The commit/PR titles and description should reflect this so they're not misleading.

[1] https://github.com/eclipse/omr/blob/00b9733a2fd7e80be2010338ff7134a381ea1310/compiler/optimizer/LoopVersioner.hpp#L1042

Reply:
ah yes, sorry about that. I have updated the commit and the PR.